docker-compose.yaml

```diff
@@ -1,4 +1,4 @@
-version: '3.7'
+﻿version: '3.7'
```

Comment:
What is this diff? :thinking: It looks identical. Could you please revert the changes to this file? Thanks.